frame/nfts/src/weights.rs

```diff
@@ -104,54 +107,54 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	/// The range of component `m` is `[0, 1000]`.
 	/// The range of component `a` is `[0, 1000]`.
 	fn destroy(n: u32, m: u32, a: u32, ) -> Weight {
-		Weight::from_ref_time(0 as u64)
-			// Standard Error: 12_000
-			.saturating_add(Weight::from_ref_time(10_091_000 as u64).saturating_mul(n as u64))
-			// Standard Error: 12_000
-			.saturating_add(Weight::from_ref_time(1_748_000 as u64).saturating_mul(m as u64))
-			// Standard Error: 12_000
-			.saturating_add(Weight::from_ref_time(1_621_000 as u64).saturating_mul(a as u64))
+		Weight::from_ref_time(2_440_842_000 as u64)
```

Comment:
:eyes:

Reply:
That seems strange, but at the same time `0` didn't seem valid either. It does not necessarily look like this weight is affected by any of the swap changes, I can't really see this PR touching any of the `destroy` logic.

Reply:
I've only seen that weight size on xcm messages but there's no xcm calls here are there?

Reply:
nope